index.html

```diff
@@ -1094,25 +1094,26 @@ <h2 class="ui header">Glyphs <em class="fontname" style="display: none;">FontNam
       
       if(t.attr('data-dis') == 0)
       {
+
         glyphs.push(
-          ' ' + (isFirstCharacter ? ' ' : ',') + '{ ' +
```

Comment:
If we don't use `isFirstCharacter` anymore, we should remove all initialisation and modification code related to it I guess

Reply:
You'r right, I removed `isFirstCharacter`